### DISCUSSION TITLE
Now when we all have trust issues with our favorite extesions I think this could help.

### DISCUSSION THREAD

**Discussion by 1Mgtrn3** [gioxx/MarvellousSuspender#88] [Ideas]
I think I don't really need any non-essential updates to an already working extension. Seriously, the consequences of it becoming malicious outweigh any of the new bells and whistles the maintainer can come up with. This extension really saves my everyday experience and I don't want to lose it once more. So I googled a **bad** solution. Here's how you can turn off updates for an extension:
https://stackoverflow.com/questions/27657617/how-to-disable-google-chrome-extension-autoupdate#:~:text=Disabling%20all%20Google%20Chrome%20updates.&text=Just%20type%20chrome%3A%2F%2Fplugins,turn%20Google%20Update%20plugin%20off.
The id of this extension is noogafoofpebimajpfpamcfhoaifemoa as of now.
If anyone has something better to offer - I would gladly hear you out.

**Comment by Ocel8** [comment]
If you install the extension from github I don't think it gets the chrome store update, but I'm not sure
